asp-core/build.gradle

```diff
@@ -1,3 +1,4 @@
 dependencies {
+     compile group: 'org.apache.commons', name: 'commons-text', version: '1.8'
```

Comment:
If am right with my coment in CryptoAccess, we do not need this dependency to solve the issue problem.